src/Aspire.Hosting.Garnet/Aspire.Hosting.Garnet.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>aspire hosting garnet</PackageTags>
+    <Description>Garnet® support for .NET Aspire.</Description>
+    <PackageIconFullPath>$(SharedDir)garnet-cube-red_white-rgb.png</PackageIconFullPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(SharedDir)VolumeNameGenerator.cs" Link="Utils\VolumeNameGenerator.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Aspire.Hosting\Aspire.Hosting.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
```

Comment:
Do you need this? I've been trying to limit the places where we use IVT so we know we are testing like customers will use the product.

Reply:
I'm using GarnetContainerImageTags in tests
<img width="472" alt="image" src="https://github.com/dotnet/aspire/assets/52016832/9936969f-8f5c-48c8-b610-c76c324bc8fc">

Reply:
I deleted it. Check if this option suits you?

But I also noticed that IVT is used in other projects.
Redis, MySql, MongoDB and others.

<img width="776" alt="image" src="https://github.com/dotnet/aspire/assets/52016832/c24c791b-64b6-4ce2-beb3-bf8467027c10">

Is it worth creating a separate PR to fix this?

Reply:
> Is it worth creating a separate PR to fix this?

If it is simple, like the case here where it is just trying to reuse constant strings, sure. I think some of them may be more complicated though.

Reply:
> If it is simple, like the case here where it is just trying to reuse constant strings, sure. I think some of them may be more complicated though.

Understood. then I'll try to do it.